src/index.ts

```diff
@@ -1,45 +1,183 @@
-function _(selector: string): any {
-	/**
-	 * innerHTML() {
-	 * }
-	 *
-	 * show() {
-	 * }
-	 *
-	 * hidden() {
-	 * }
-	 *
-	 * addEvent() {
-	 * }
-	 */
+import { CustomNode, FetchOptions, FetchResponse } from "./@types";
+
+function _(selector: string): CustomNode {
+  const element = document.querySelector(selector);
+
+  if (element === null)
+    throw new Error("Error: there is no element with entered selector.");
+
+  const get = () => element;
+
+  const innerHTML = (value: string) => {
+    if (value) element.innerHTML = value;
+
+    return element.innerHTML;
+  };
+
+  const show = () => {
+    if (element instanceof HTMLElement) {
+      element.style.display = "block";
+    }
+  };
+
+  const hidden = () => {
+    if (element instanceof HTMLElement) element.style.display = "none";
+  };
+
+  const addEvent = <K extends keyof HTMLElementEventMap>(
+    type: K,
+    listener: (this: HTMLElement, ev: HTMLElementEventMap[K]) => any
+  ) => {
+    if (element instanceof HTMLElement)
+      element.addEventListener(type, listener);
+  };
+
+  return {
+    get,
+    innerHTML,
+    show,
+    hidden,
+    addEvent,
+  };
 }
 
 module _ {
-	export function fetch() {
-		return {};
-	}
+  export function fetch<T = any>(
+    url: string,
+    options?: FetchOptions
+  ): Promise<FetchResponse<T>> {
+    if (options !== undefined) return fetch(url, { ...options });
+    return fetch(url);
+  }
+
+  export function isNull(value: unknown) {
```

Comment:
이 함수와 아래 함수들의 현재 추론된 반환 타입은 `boolean`인데 좀 더 좁혀준다면 타입 가드를 할 때 활용할 수도 있을 것 같네요! [LMS 자료](https://techcourse.woowahan.com/s/idNrOtc7/ls/lOqYjIvN)

Reply:
좀 더 좁혀준다는 말씀이 잘 이해가 가지 않는데요!

```suggestion
  export function isNull<T extends unknown>(value: T) {
```
로 사용해서 실제로 `isNull`을 호출하는 자리에서 `isNull<HTMLElement>(element)` 로 사용한다는 점에서 의미가 있다는 말씀일까요?

Reply:
아니면.. is 사용?
```suggestion
  export function isNull(value: unknown): value is number{
```